docs/user/rstudio/index.qmd

```diff
@@ -49,16 +50,19 @@ Posit Workbench administrators can reference the [Posit Workbench Administrator
 
 ### Open Source
 
-- [RStudio Desktop download](https://www.rstudio.com/products/rstudio/download/) - available for Windows/Mac/Linux
-- [RStudio Server download](https://www.rstudio.com/products/rstudio/download-server/) - available for Linux
+- [RStudio Desktop download](https://posit.co/downloads/) - available for Windows/Mac/Linux
+- [RStudio Server download](https://posit.co/downloads/) - available for Linux
 
 ### Commercial
 
 - [Posit Workbench](https://docs.rstudio.com/rsw/installation/) - administration guide for installation and configuration
-- [RStudio Desktop Pro](https://www.rstudio.com/products/rstudio-desktop-pro/download-commercial/) - available for Windows/Mac/Linux
+- [RStudio Desktop Pro](https://posit.co/downloads/) - available for Windows/Mac/Linux
```

Comment:
Should the link here be `https://posit.co/downloads/` or `https://posit.co/download/rstudio-desktop-pro/` ?